pyiode/lec.pyx

```diff
@@ -12,6 +12,8 @@ from lec cimport IodeExecLec, IodeExecLecT
 from iode_python cimport SCR_free
 
 
+# GDM> I think there should be some kind of Workspace singleton class (which would contain a few AbstractDatabase attributes)
```

Comment:
You mean a workspace singleton class for each IODE type ? A special separated Workspace (Database) class ? I don't follow you.